### PR TITLE
initial configuration of lavamoat allow scripts

### PR DESCRIPTION
**Pull request by sethkfman**
**Description**

Adding Lavamoat Allow-Scripts to the project

**Checklist**

* [X] There is a related GitHub issue
* [X] Tests are included if applicable
* [X] Any added code is fully documented

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "clean:ios": "rm -rf ios/build",
     "pod:install": "command -v pod && (cd ios/ && pod install && cd ..) || echo \"pod command not found\"",
     "clean:android": "rm -rf android/app/build",
-    "clean:node": "rm -rf node_modules && yarn install",
+    "clean:node": "rm -rf node_modules && yarn install && yarn allow-scripts",
```
Comment (kumavis) [inline package.json:12]: - add a "setup" script, containing something like "yarn install && yarn patch-package && yarn allow-scripts"
- use "yarn setup" in place of install in "clean:node"

File: package.json [hunk 2]
```diff
@@ -336,5 +338,21 @@
     "assets": [
       "./app/fonts"
     ]
+  },
+  "lavamoat": {
+    "allowScripts": {
+      "detox": true,
+      "@sentry/cli": true,
```
Comment (kumavis) [inline package.json:345]: disable these if possible

File: package.json [hunk 3]
```diff
@@ -336,5 +338,21 @@
...
+      "@lavamoat/preinstall-always-fail": false,
+      "core-js": false,
+      "core-js-pure": false,
+      "dtrace-provider": false,
+      "fsevents": false,
+      "husky": false,
```
Comment (kumavis) [inline package.json:351]: if using husky, this might break it

**Comment by rickycodes** [comment]
closin in favor of #3367

**Pull request closed without merging**